lib/getatrex/collector.ex

```diff
@@ -84,11 +102,142 @@ defmodule Getatrex.Collector do
     {:reply, :ok, Map.put(state, :msgstr, translated_string)}
   end
 
+# msgstr[5]
+  def handle_call({:dispatch_line, ~s(msgstr[5] "") <> _tail}, _from, %{msgid_plural: msgid_plural} = state) do
+    translated_string =
```

Comment:
@guidotripaldi Love the PR, thanks a lot! 

One thing though I would like to have improved - make this one more DRY. 
For example, extract `translated_string` into a reusable function for those 5 call handlers.  

and/or 

patterns match `~s(msgstr[) <> tail` where the `tail` will start with a number (as string) which could be also pattern matched. 

Please let me know if you have more questions.